FV3/atmos_cubed_sphere/model/dyn_core.F90

```diff
@@ -973,6 +989,15 @@ subroutine dyn_core(npx, npy, npz, ng, sphum, nq, bdt, n_split, zvir, cp, akap,
 #ifdef USE_COND
      call complete_group_halo_update(i_pack(11), domain)
 #endif
+
+        
+    !$ser savepoint HaloUpdate-Out
+    !$ser data array=pt
+     
+    !$ser savepoint HaloUpdate-Out
+    !$ser data array=delp
```

Comment:
For my understanding of Serialbox, why do these go on separate lines?

Reply:
Oh in this case I have multiple instances of the same unit test -- for every input we need a corresponding output to make the test code simple. I could have combined the two arrays into a HaloUpdate2Vars variable or something, or we could remove one of these (just have them both because they were part of the same 'i_pack' ). Every savepoint gets a unique id, so these are two different savepoints.

Reply:
How does `HaloUpdate-Out` translate to an ID?

Reply:
It will get a unique id as part of the savepoint (basically an incremental counter), which will be stored in the Metadata json and the test will pick up on it